### PR TITLE
Updated wallet strings #19940

### PR DESCRIPTION
**Pull request by jamesmudgett**
Found inconsistencies in wallet strings from desktop and iOS

Resolves https://github.com/brave/brave-browser/issues/19940

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [x] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: browser/ui/android/strings/android_brave_strings.grd [hunk 1]
```diff
@@ -1738,43 +1738,43 @@ Are you sure you want to do this?
...
     <message name="IDS_SETUP_CRYPTO" desc="Brave Wallet onboarding button text">
-      Setup Crypto
+      Get started
     </message>
     <message name="IDS_BACKUP_YOUR_WALLET" desc="Brave Wallet onboarding text">
       Backup your wallet now!
     </message>
     <message name="IDS_BACKUP_YOUR_WALLET_TEXT" desc="Brave Wallet onboarding text">
-      In the next step you will see 12 words that allow you to recover your crypto wallet.
+      In the next step you’ll see a 12-word recovery phrase, which you can use to recover your primary crypto accounts. Safe it someplace safe. Your recovery phrase is the only way to regain account access in case of forgotten password, lost or stolen device, or you want to switch wallets.
```
Comment (SergeyZhukovsky) [inline browser/ui/android/strings/android_brave_strings.grd:1753]: you should use ' char instead of `, it will fail otherwise.

Comment (jamesmudgett) [reply browser/ui/android/strings/android_brave_strings.grd]: fixed

File: browser/ui/android/strings/android_brave_strings.grd [hunk 2]
```diff
@@ -1738,43 +1738,43 @@ Are you sure you want to do this?
...
     <message name="IDS_BACKUP_YOUR_WALLET_TEXT" desc="Brave Wallet onboarding text">
-      In the next step you will see 12 words that allow you to recover your crypto wallet.
+      In the next step you'll see a 12-word recovery phrase, which you can use to recover your primary crypto accounts. Safe it someplace safe. Your recovery phrase is the only way to regain account access in case of forgotten password, lost or stolen device, or you want to switch wallets.
     </message>
     <message name="IDS_CONTINUE_TEXT" desc="Brave Wallet onboarding text">
       Continue
     </message>
     <message name="IDS_BACKUP_WALLET_CHECKBOX_TEXT" desc="Brave Wallet onboarding text">
-      I understand that if I lose my recovery words, I will not be able to access my crypto wallet.
+      I understand that if I lose my recovery phrase, I won’t be able to access my crypto wallet.
```
Comment (SergeyZhukovsky) [inline browser/ui/android/strings/android_brave_strings.grd:1759]: `won’t` --> `won't`

Comment (SergeyZhukovsky) [reply browser/ui/android/strings/android_brave_strings.grd:1759]: CI has passed, I think we are ok, we had troubles with that in the past, maybe something changed

**Review by SergeyZhukovsky — APPROVED** [review]
++

File: browser/ui/android/strings/android_brave_strings.grd [hunk 3]
```diff
@@ -1738,43 +1738,43 @@ Are you sure you want to do this?
...
     <message name="IDS_SETUP_CRYPTO" desc="Brave Wallet onboarding button text">
-      Setup Crypto
+      Get started
     </message>
     <message name="IDS_BACKUP_YOUR_WALLET" desc="Brave Wallet onboarding text">
       Backup your wallet now!
     </message>
     <message name="IDS_BACKUP_YOUR_WALLET_TEXT" desc="Brave Wallet onboarding text">
-      In the next step you will see 12 words that allow you to recover your crypto wallet.
+      In the next step you'll see a 12-word recovery phrase, which you can use to recover your primary crypto accounts. Safe it someplace safe. Your recovery phrase is the only way to regain account access in case of forgotten password, lost or stolen device, or you want to switch wallets.
```
Comment (bbondy) [inline browser/ui/android/strings/android_brave_strings.grd:1753]: *Save it someplace safe

File: browser/ui/android/strings/android_brave_strings.grd [hunk 4]
```diff
@@ -1738,43 +1738,43 @@ Are you sure you want to do this?
...
-      Write down or copy these words in the right order and save them somewhere safe.
+      Write down or copy these words in the exact order shown below, and save them somewhere safe. Your recovery phrase is the only way to regain account access in case of forgotten password, lost or stolen device, or you want to switch wallets.
     </message>
     <message name="IDS_YOUR_RECOVERY_PHRASE_WARNING_TEXT" desc="Brave Wallet onboarding text">
-      WARNING: Never disclose your backup phrase. Anyone with this phrase can take your funds forever.
+      WARNING: Never share your recovery phrase. Anyone with this phrase can take your assets forever.
     </message>
     <message name="IDS_RECOVERY_PHRASE_CHECKBOX_TEXT" desc="Brave Wallet onboarding text">
-      I have backed up my phrase somewhere safe
+      I have backed up my phrase somewhere safe.
```
Comment (bbondy) [inline browser/ui/android/strings/android_brave_strings.grd:1771]: want a period here?

File: browser/ui/android/strings/android_brave_strings.grd [hunk 5]
```diff
@@ -1792,13 +1792,13 @@ Are you sure you want to do this?
       Enter Password
     </message>
     <message name="IDS_RETYPE_PASSWORD" desc="Brave wallet password hint text">
-      Re-enter password
+      Verify password
     </message>
     <message name="IDS_PASSWORD_ERROR" desc="Brave Wallet password error text">
-      Password must be 7 or more characters
+      Passwords must be at least 7 characters, and contain at least one number and one special character.
```
Comment (bbondy) [inline browser/ui/android/strings/android_brave_strings.grd:1798]: also must have one letter

File: browser/ui/android/strings/android_brave_strings.grd [hunk 6]
```diff
@@ -1960,7 +1960,7 @@ Are you sure you want to do this?
       To
     </message>
     <message name="IDS_TO_ADDRESS_EDIT" desc="Brave Wallet buy-send-swap section for address(hint)">
-      address or url
+      Address or url
```
Comment (bbondy) [inline browser/ui/android/strings/android_brave_strings.grd:1963]: URL

**Review by bbondy — CHANGES_REQUESTED** [review]

File: browser/ui/android/strings/android_brave_strings.grd (hunk 4 above)
Comment (jamesmudgett) [reply browser/ui/android/strings/android_brave_strings.grd:1771]: Yes, same as on desktop

File: browser/ui/android/strings/android_brave_strings.grd (hunk 5 above)
Comment (jamesmudgett) [reply browser/ui/android/strings/android_brave_strings.grd]: Updated from slack

**Review by bbondy — APPROVED** [review]

**Pull request merged**